--- clap-2.24.2/Cargo.toml	2017-05-16T13:28:18+02:00
+++ clap-2.24.2/Cargo.toml	2017-06-15T18:15:07.175214+02:00
@@ -16,7 +16,7 @@
 """
 
 [dependencies]
-bitflags              = "0.8.0"
+bitflags              = "0.9"
 vec_map               = "0.8"
 unicode-width         = "0.1.4"
 unicode-segmentation  = "1.0.1"
@@ -24,7 +24,6 @@
 ansi_term = { version = "0.9.0",  optional = true }
 term_size = { version = "0.3.0",  optional = true }
 yaml-rust = { version = "0.3.5",  optional = true }
-clippy    = { version = "~0.0.131", optional = true }
 atty      = { version = "0.2.2",  optional = true }
 
 [dev-dependencies]
@@ -38,8 +37,6 @@
 wrap_help   = ["term_size"]
 yaml        = ["yaml-rust"]
 unstable    = [] # for building with unstable clap features (doesn't require nightly Rust) (currently none)
-nightly     = [] # for building with unstable Rust features (currently none)
-lints       = ["clippy"] # Requires nightly Rust
 debug       = [] # Enables debug messages
 no_cargo    = [] # Enable if you're not using Cargo, disables Cargo-env-var-dependent macros
 
