--- clap-2.20.5/Cargo.toml	2017-02-18T17:43:08+01:00
+++ clap-2.20.5/Cargo.toml	2017-02-28T12:59:42.854289+01:00
@@ -16,8 +16,8 @@
 """
 
 [dependencies]
-bitflags              = "0.7.0"
-vec_map               = "0.6.0"
+bitflags              = "0.8.0"
+vec_map               = "0.7.0"
 unicode-width         = "0.1.4"
 unicode-segmentation  = "1.0.1"
 strsim    = { version = "0.6.0",  optional = true }
@@ -25,7 +25,6 @@
 term_size = { version = "0.2.2",  optional = true }
 libc      = { version = "0.2.20",  optional = true }
 yaml-rust = { version = "0.3.5",  optional = true }
-clippy    = { version = "~0.0.112", optional = true }
 
 [dev-dependencies]
 regex = "~0.1.80"
@@ -38,7 +37,6 @@
 yaml        = ["yaml-rust"]
 unstable    = [] # for building with unstable clap features (doesn't require nightly Rust) (currently none)
 nightly     = [] # for building with unstable Rust features (currently none)
-lints       = ["clippy"] # Requires nightly Rust
 debug       = [] # Enables debug messages
 no_cargo    = [] # Enable if you're not using Cargo, disables Cargo-env-var-dependent macros
 
