--- num-0.1.37/Cargo.toml	2017-02-26T23:17:39+01:00
+++ num-0.1.37/Cargo.toml	2017-03-02T10:24:09.640914+01:00
@@ -20,30 +20,24 @@
 
 [dependencies.num-bigint]
 optional = true
-path = "bigint"
 version = "0.1.36"
 
 [dependencies.num-complex]
 optional = true
-path = "complex"
 version = "0.1.36"
 
 [dependencies.num-integer]
-path = "./integer"
 version = "0.1.33"
 
 [dependencies.num-iter]
 optional = false
-path = "iter"
 version = "0.1.33"
 
 [dependencies.num-rational]
 optional = true
-path = "rational"
 version = "0.1.36"
 
 [dependencies.num-traits]
-path = "./traits"
 version = "0.1.37"
 
 [dev-dependencies]
