--- num-0.1.39/Cargo.toml	2017-06-09T20:03:08+02:00
+++ num-0.1.39/Cargo.toml	2017-06-15T16:34:59.207266+02:00
@@ -20,30 +20,24 @@
 
 [dependencies.num-bigint]
 optional = true
-path = "bigint"
 version = "0.1.39"
 
 [dependencies.num-complex]
 optional = true
-path = "complex"
 version = "0.1.38"
 
 [dependencies.num-integer]
-path = "./integer"
 version = "0.1.34"
 
 [dependencies.num-iter]
 optional = false
-path = "iter"
 version = "0.1.33"
 
 [dependencies.num-rational]
 optional = true
-path = "rational"
 version = "0.1.38"
 
 [dependencies.num-traits]
-path = "./traits"
 version = "0.1.39"
 
 [dev-dependencies]
