--- num-rational-0.1.38/Cargo.toml	2017-06-09T19:58:22+02:00
+++ num-rational-0.1.38/Cargo.toml	2017-06-15T16:11:19.626928+02:00
@@ -13,15 +13,12 @@
 
 [dependencies.num-bigint]
 optional = true
-path = "../bigint"
 version = "0.1.32"
 
 [dependencies.num-integer]
-path = "../integer"
 version = "0.1.32"
 
 [dependencies.num-traits]
-path = "../traits"
 version = "0.1.32"
 
 [dependencies.rustc-serialize]
