--- ignore-0.1.8/Cargo.toml	2017-03-13T02:32:35+01:00
+++ ignore-0.1.8/Cargo.toml	2017-03-15T12:24:59.102230+01:00
@@ -19,7 +19,7 @@
 
 [dependencies]
 crossbeam = "0.2"
-globset = { version = "0.1.4", path = "../globset" }
+globset = { version = "0.1.4" }
 lazy_static = "0.2"
 log = "0.3"
 memchr = "1"
@@ -30,8 +30,5 @@
 [dev-dependencies]
 tempdir = "0.3.5"
 
-[features]
-simd-accel = ["globset/simd-accel"]
-
 [profile.release]
 debug = true
