--- ignore-0.2.0/Cargo.toml	2017-05-12T01:07:08+02:00
+++ ignore-0.2.0/Cargo.toml	2017-06-14T16:15:10.677802+02:00
@@ -19,7 +19,7 @@
 
 [dependencies]
 crossbeam = "0.2"
-globset = { version = "0.2.0", path = "../globset" }
+globset = { version = "0.2.0" }
 lazy_static = "0.2"
 log = "0.3"
 memchr = "1"
@@ -30,8 +30,5 @@
 [dev-dependencies]
 tempdir = "0.3.5"
 
-[features]
-simd-accel = ["globset/simd-accel"]
-
 [profile.release]
 debug = true
