From d489d3721cafbbdfd05071d3ce608acfbe5a8587 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <ignatenko@redhat.com>
Date: Sun, 26 Feb 2017 15:09:09 +0100
Subject: [PATCH 2/3] bump regex to 0.2

Signed-off-by: Igor Gnatenko <ignatenko@redhat.com>
---
 Cargo.toml    | 2 +-
 src/string.rs | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index a5d802d..e358fbf 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,7 +18,7 @@ cargo-fmt = []
 toml = "0.2.1"
 rustc-serialize = "0.3"
 unicode-segmentation = "1.0.0"
-regex = "0.1"
+regex = "0.2"
 term = "0.4"
 strings = "0.0.1"
 diff = "0.1"
diff --git a/src/string.rs b/src/string.rs
index 71d8f84..f8714ad 100644
--- a/src/string.rs
+++ b/src/string.rs
@@ -33,7 +33,7 @@ pub struct StringFormat<'a> {
 // FIXME: simplify this!
 pub fn rewrite_string<'a>(orig: &str, fmt: &StringFormat<'a>) -> Option<String> {
     // Strip line breaks.
-    let re = Regex::new(r"([^\\](\\\\)*)\\[\n\r][:space:]*").unwrap();
+    let re = Regex::new(r"([^\\](\\\\)*)\\[\n\r][[:space:]]*").unwrap();
     let stripped_str = re.replace_all(orig, "$1");
 
     let graphemes = UnicodeSegmentation::graphemes(&*stripped_str, false).collect::<Vec<&str>>();
-- 
2.11.1

