From 3c35b075f9aa52a1f76f120741531ebb2b0b0121 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <ignatenko@redhat.com>
Date: Sun, 26 Feb 2017 15:41:32 +0100
Subject: [PATCH 3/3] bump syntex_* to 0.58

Signed-off-by: Igor Gnatenko <ignatenko@redhat.com>
---
 Cargo.toml      |  4 ++--
 src/patterns.rs | 13 ++++++++++---
 2 files changed, 12 insertions(+), 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index e358fbf..c3d8ee2 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,8 +22,8 @@ regex = "0.2"
 term = "0.4"
 strings = "0.0.1"
 diff = "0.1"
-syntex_syntax = "0.56"
-syntex_errors = "0.56"
+syntex_syntax = "0.58"
+syntex_errors = "0.58"
 log = "0.3"
 env_logger = "0.4"
 getopts = "0.2"
diff --git a/src/patterns.rs b/src/patterns.rs
index 03598a1..335ea98 100644
--- a/src/patterns.rs
+++ b/src/patterns.rs
@@ -18,7 +18,7 @@ use types::{rewrite_path, PathContext};
 use super::Spanned;
 use comment::FindUncommented;
 
-use syntax::ast::{self, BindingMode, Pat, PatKind, FieldPat};
+use syntax::ast::{self, BindingMode, Pat, PatKind, FieldPat, RangeEnd};
 use syntax::ptr;
 use syntax::codemap::{self, BytePos, Span};
 
@@ -54,8 +54,15 @@ impl Rewrite for Pat {
                     None
                 }
             }
-            PatKind::Range(ref lhs, ref rhs) => {
-                rewrite_pair(&**lhs, &**rhs, "", "...", "", context, width, offset)
+            PatKind::Range(ref lhs, ref rhs, ref end_kind) => {
+                match *end_kind {
+                    RangeEnd::Included => {
+                        rewrite_pair(&**lhs, &**rhs, "", "...", "", context, width, offset)
+                    }
+                    RangeEnd::Excluded => {
+                        rewrite_pair(&**lhs, &**rhs, "", "..", "", context, width, offset)
+                    }
+                }
             }
             PatKind::Ref(ref pat, mutability) => {
                 let prefix = format!("&{}", format_mutability(mutability));
-- 
2.11.1

