From 2e39c53b21f4cd957a620149e0388af7013e2d17 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <ignatenko@redhat.com>
Date: Sat, 25 Feb 2017 23:42:28 +0100
Subject: [PATCH] bump serde to 0.9

Signed-off-by: Igor Gnatenko <ignatenko@redhat.com>
---
 Cargo.toml | 6 +++---
 src/lib.rs | 5 +++--
 2 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 8017ac3..4ffcc3d 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,8 +37,8 @@ keywords = ["data-structures", "collections", "vecmap", "vec_map", "contain-rs"]
 readme = "README.md"
 
 [features]
-eders = ["serde", "serde_macros"]
+eders = ["serde", "serde_derive"]
 
 [dependencies]
-serde = { version = "0.6", optional = true }
-serde_macros = { version = "0.6", optional = true }
+serde = { version = "0.9", optional = true }
+serde_derive = { version = "0.9", optional = true }
diff --git a/src/lib.rs b/src/lib.rs
index 3af7cfd..8958ae1 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -14,10 +14,11 @@
 //! are O(highest integer key).
 
 // optional serde support
-#![cfg_attr(feature = "eders", feature(const_fn, custom_derive, plugin))]
-#![cfg_attr(feature = "eders", plugin(serde_macros))]
 #[cfg(feature = "eders")]
 extern crate serde;
+#[cfg(feature = "eders")]
+#[macro_use]
+extern crate serde_derive;
 
 use self::Entry::*;
 
-- 
2.11.1

