--- syntex_syntax-0.58.0/Cargo.toml	2017-01-27T23:23:52+01:00
+++ syntex_syntax-0.58.0/Cargo.toml	2017-02-18T17:15:12.797226+01:00
@@ -8,9 +8,9 @@
 include = ["Cargo.toml", "src/**/*.rs"]
 
 [dependencies]
-bitflags = "0.7.0"
+bitflags = "0.8.0"
 log = "0.3.6"
 rustc-serialize = "0.3.16"
-syntex_errors = { version = "0.58", path = "../syntex_errors" }
-syntex_pos = { version = "0.58", path = "../syntex_pos" }
+syntex_errors = { version = "0.58" }
+syntex_pos = { version = "0.58" }
 unicode-xid = "0.0.4"
