--- syntex_syntax-0.58.1/Cargo.toml	2017-02-19T22:20:01+01:00
+++ syntex_syntax-0.58.1/Cargo.toml	2017-02-24T22:27:05.554920+01:00
@@ -12,6 +12,6 @@
 bitflags = "0.8.0"
 log = "0.3.6"
 rustc-serialize = "0.3.16"
-syntex_errors = { version = "0.58", path = "../syntex_errors" }
-syntex_pos = { version = "0.58", path = "../syntex_pos" }
+syntex_errors = { version = "0.58" }
+syntex_pos = { version = "0.58" }
 unicode-xid = "0.0.4"
