--- ripgrep-0.5.0/Cargo.toml	2017-03-13T03:32:21+01:00
+++ ripgrep-0.5.0/Cargo.toml	2017-03-15T12:32:53.469519+01:00
@@ -31,8 +31,8 @@
 clap = "2.20.5"
 encoding_rs = "0.5.0"
 env_logger = { version = "0.4", default-features = false }
-grep = { version = "0.1.5", path = "grep" }
-ignore = { version = "0.1.7", path = "ignore" }
+grep = { version = "0.1.5" }
+ignore = { version = "0.1.7" }
 lazy_static = "0.2"
 libc = "0.2"
 log = "0.3"
@@ -41,15 +41,11 @@
 num_cpus = "1"
 regex = "0.2.1"
 same-file = "0.1.1"
-termcolor = { version = "0.3.0", path = "termcolor" }
+termcolor = { version = "0.3.0" }
 
 [build-dependencies]
 clap = "2.18"
 lazy_static = "0.2"
 
-[features]
-avx-accel = ["bytecount/avx-accel"]
-simd-accel = ["bytecount/simd-accel", "regex/simd-accel"]
-
 [profile.release]
 debug = true
