--- encoding_rs-0.6.11/Cargo.toml	2017-06-02T11:00:53+02:00
+++ encoding_rs-0.6.11/Cargo.toml	2017-06-15T17:39:38.438464+02:00
@@ -15,14 +15,12 @@
 travis-ci = { repository = "hsivonen/encoding_rs" }
 
 [features]
-simd-accel = ["simd"]
 no-static-ideograph-encoder-tables = []
 parallel-utf8 = ["rayon"]
 
 [dependencies]
 cfg-if = "0.1.0"
-simd = { version = "0.2.0", optional = true }
-rayon = { version = "0.7.0", optional = true }
+rayon = { version = "0.8.0", optional = true }
 
 [profile.release]
 lto = true
