--- serde_derive-0.9.11/Cargo.toml	2017-03-06T01:51:24+01:00
+++ serde_derive-0.9.11/Cargo.toml	2017-03-06T08:55:23.189648+01:00
@@ -23,5 +23,5 @@
 
 [dependencies]
 quote = "0.3.8"
-serde_codegen_internals = { version = "=0.14.1", default-features = false, path = "../serde_codegen_internals" }
+serde_codegen_internals = { version = "=0.14.1", default-features = false }
 syn = { version = "0.11", features = ["visit"] }
